helm/kadalu/charts/operator/templates/pvc.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kadalu-sts-log-pvc
+  namespace: {{ .Release.Namespace }}
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Mi
+  volumeName: kadalu-sts-log-pv
+  selector:
+    matchLabels:
+      type: kadalu-sts
+---  
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kadalu-ds-log-pvc
+  namespace: {{ .Release.Namespace }}
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Mi
```

Comment:
Is this enough? Is there log rotation?

Reply:
No. There is no log rotation. Can we make those part of  another task?

Reply:
Well actually, a step back why do we need a log volume at all?
Kadalu, like most k8s components, logs to the console and the container logs are captured there.